dali/ft/filecopy.cpp

```diff
@@ -3154,21 +3154,24 @@ void FileSprayer::updateTargetProperties()
 
                     curProps.setProp("@modified", temp.getString(timestr).str());
                 }
-                if (replicate && (distributedSource != distributedTarget) )
+                if ((distributedSource != distributedTarget) && (cur.whichInput != (unsigned)-1))
```

Comment:
When can cur.whichInput be -1 ?

Reply:
When there is no (physical) input file like superfile/-key copy or spray empty directory.